packages/from-schema/src/index.ts

```diff
@@ -147,12 +147,20 @@ const run: (schemaInput: GraphQLSchema, optionsInput: IInternalOptions) => strin
       );
     }
 
+    // Becuase enums when declared do not end in semicolons and types do
+    // the we wrap the format enum function depending on if we are using the
+    // type builder or the enum type number
+    const wrappedEnumFormatFunction: () => string = function (): string {
```

Comment:
Let's remove the word `Function` from the name, and use an arrow function instead

Reply:
Can we have this take arguments instead of relying on the variables from the outerscope?